nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4570,18 +4570,18 @@ async fn rack_view(
 /// List uninitialized sleds in a given rack
 #[endpoint {
     method = GET,
-    path = "/v1/system/hardware/uninitialized-sleds",
+    path = "/v1/system/hardware/sleds-uninitialized",
     tags = ["system/hardware"]
 }]
-async fn uninitialized_sled_list(
+async fn sled_list_uninitialized(
     rqctx: RequestContext<Arc<ServerContext>>,
```

Comment:
this is insufficient to make this into a paginated endpoint. We also need the `Query<..>` even though we'll ignore it.

Reply:
Sorry about that and thanks for the tip! I will update it in a follow up.

Reply:
I'm sorry I didn't get this comment in in time!

Reply:
Fixed in https://github.com/oxidecomputer/omicron/pull/4720